README.md

```diff
@@ -9,46 +9,48 @@
+### Our example will be: The WhatsApp Web Application!
 
 ## Server Setup (Attacker's hosting):
-1. Upload "qrHandler.php" to your server, this php file is used to convert the base64 qr code string into a valid .JPG file
+1. Upload "qrHandler.php" to your server. This php file is used to convert the base64 QR Code string into a valid .JPG file.
 
-	Now we have a valid generated QR image named "tmp.jpg" residing in the same root folder of your files and will be updated whenever that php file will be called, So we can put it anywhere "for example a fake WhatsApp page, a scam page with an offer related to WhatsApp, etc... depending on your creativity"
+	Now you have a valid, generated, QR image named "tmp.jpg" residing in the same root folder as your files which will be updated whenever that php file is called, so we can put it anywhere. For example: a fake WhatsApp page, a scam page with an offer related to WhatsApp, etc. Depending on your creativity.
 
-2. Now update the "phishing.html" file your prefered phishing page source code.
+2. Now update the "phishing.html" file with your prefered phishing page source code.
 
 
 ## Client Side Setup (Attacker's browser):
 
-1. Open your Firefox browser!
-2. Write "about:config" in the url area, click the "i'll be careful, i promise" confirmation button.
-3. Search for a preference named "security.csp.enable" and change it's value to "false" by double clicking it to allow performing an XHR Request over a different domain (We're not supporting leaving this preference disabled, you may do that while testing, but after that you should set the preference to its original state).
-4. Instal Greasemonkey addon (https://addons.mozilla.org/en-US/firefox/addon/greasemonkey) and be sure that the module file "WhatsAppQRJackingModule.js" is loaded and already running!
-5. Now We're Ready, Browse to our example "https://web.whatsapp.com" on your side, Wait for a WhatsApp session to be loaded, Greasemonkey should now inject our WhatsApp module file to catch and  .
-6. Send the direct link of the final phishing page to a victim "Once the QR scanned, Victim's session is yours now"
+1. Open your Firefox browser.
+2. Write "about:config" in the url area, click the "I'll be careful, I promise" confirmation button.
+3. Search for a preference named "security.csp.enable" and change it's value to "false" by double clicking it to allow performing an XHR Request from a different domain (we're not supporting leaving this preference disabled, you may do that while testing, but after that you should set the preference to its original state).
+4. Install the Greasemonkey addon (https://addons.mozilla.org/en-US/firefox/addon/greasemonkey) and be sure that the module file "WhatsAppQRJackingModule.js" is loaded and already running.
+5. Now we're ready. Browse to our example "https://web.whatsapp.com" on your side and wait for a WhatsApp session to be loaded. Greasemonkey should now inject our WhatsApp module file to catch.
```

Comment:
I didn't understand the original, it said: "to catch and ." 

Was the supposed to be there? I removed the "and ." because I was unsure if anything was supposed to be written after.